src/common/paths/buckets.yml

```
@@ -3,49 +3,120 @@ get:
+              "self": "/api/v2/buckets?descending=false\u0026limit=1\u0026offset=50\u0026orgID=ORG_ID"
+          },
+          "buckets": []
+        }
+      ```
+
+    #### Related Guides
+
+    - [Manage buckets]({{% INFLUXDB_DOCS_URL %}}/cloud/organizations/buckets/)
   parameters:
     - $ref: "../parameters/TraceSpan.yml"
     - $ref: "../parameters/Offset.yml"
     - $ref: "../parameters/Limit.yml"
     - $ref: "../parameters/After.yml"
     - in: query
       name: org
-      description: The name of the organization.
+      description: |
+        Organization name.
+        The name of the organization.
       schema:
         type: string
     - in: query
       name: orgID
-      description: The organization ID.
+      description: |
+        Organization ID.
+        The organization ID.
       schema:
         type: string
     - in: query
       name: name
-      description: Only returns buckets with a specific name.
+      description: |
+        Bucket name.
+        Only returns buckets with a specific name.
       schema:
         type: string
     - in: query
       name: id
-      description: Only returns buckets with a specific ID.
+      description: |
+        Bucket ID.
+        Only returns buckets with a specific ID.
       schema:
         type: string
   responses:
     "200":
-      description: A list of buckets
+      description: |
+        Success.
+        The response body contains a list of buckets and URI links to adjacent
+        buckets.
       content:
         application/json:
           schema:
             $ref: "../schemas/Buckets.yml"
+          examples:
+            $ref: "../responses/example-responses/GetBucketsSuccessResponse.yml"
+    "401":
+      $ref: "../responses/AuthorizationError.yml"
+    "500":
+      $ref: "../responses/InternalServerError.yml"
     default:
       description: Unexpected error
       content:
         application/json:
           schema:
             $ref: "../schemas/Error.yml"
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl --request GET "https://cloud2.influxdata.com/api/v2/buckets?name=_monitoring" \
+          --header "Authorization: Token INFLUX_TOKEN" \
+          --header "Accept: application/json" \
+          --header "Content-Type: application/json"
 post:
   operationId: PostBuckets
   tags:
     - Buckets
   summary: Create a bucket
+  description: |
+    Creates a [bucket]({{% INFLUXDB_DOCS_URL %}}/cloud/reference/glossary/#bucket)
+    and returns the created bucket along with metadata. The default data
+    [retention period]({{% INFLUXDB_DOCS_URL %/cloud/reference/glossary/#retention-period}})
+    is 30 days.
+
+    #### InfluxDB OSS
+
+    - A single InfluxDB OSS instance supports approximately 20 buckets
+    actively being written to or queried across all organizations depending on
+    the use case. Any more than that can adversely affect performance.
```

```suggestion
    - A single InfluxDB OSS instance supports
    active writes or queries for approximately 20 buckets across all organizations at a given time. Reading or writing to more than 20 buckets at a time can adversely affect performance.
```
@kelseiv could say this better.